client/header/activity-panel/panels/inbox/card.js

```diff
@@ -3,7 +3,6 @@
  */
 import { Component } from '@wordpress/element';
 import PropTypes from 'prop-types';
-import Gridicon from 'gridicons';
```

Comment:
I see `<GridIcon />` still being used in this file and some icon related settings here which prevents my page from loading.

Following this down, I notice that icon is still being used in `ActivityCard`.  Just making sure that this is only meant to affect inbox notes and not other activity cards, is that right?

Reply:
Another note on this- the placeholder for `ActivityCard` has an "icon" div.  Not sure if we plan on leaving this or not for other panels using the activity card.

Ideally, we could pass a boolean `icon` prop which would determine if the placeholder shows this or not.

Reply:
Hi @joshuatf,

I think that after merging the [PR 4256](https://github.com/woocommerce/woocommerce-admin/pull/4256) with `add/4099` we will have the missing elements that are blocking the normal behavior of this branch. For this purpose, I created the test_branch (`add/4099_test_branch`) that has all the changes together.

>Following this down, I notice that icon is still being used in ActivityCard. Just making sure that this is only meant to affect inbox notes and not other activity cards, is that right?

Yes, exactly.
I tried to decouple the `InboxNoteCard` from the `ActivityCard`. We still continue using the `ActivityCard` with orders and reviews but not with inbox notifications anymore.
My plan was trying to have an `InboxNoteCard` that we could use as an ActivityCard in the InboxPanel or eventually in the dashboard with no big modifications.

`Gridicon` is not used anymore In the `card.js`.

>Another note on this- the placeholder for ActivityCard has an "icon" div. Not sure if we plan on leaving this or not for other panels using the activity card.

This is a good point. I left the `ActivityCardPlaceholder` -when is loading- and an empty `ActivityCard` -when we don't have results- in the inbox panel, to follow the expected behavior for the inbox panel. In the Home screen it's different. If we don't have results the whole inbox messages column will disappear. For the loading situation, I wasn't sure if we will use something specific or the same `ActivityCardPlaceholder`.

I hope have responded to all the questions you had.

If you think we should change the approach just let me know and I will change it 🙂